pages/register.js

```diff
@@ -47,10 +60,87 @@ const RegistrationForm = () => {
                         justifyItems="start"
                         overflowY="auto"
                     >
-                        <FieldSection title='First Name' />
-                        <SystemComponent>as</SystemComponent>
-                        <SystemComponent>as</SystemComponent>
+                        <FieldSection 
+                        title='Full Name '
+                        asterik={true}                     
+                            name="fullName"
+                            value={formValues['fullName']}
+                            onChange={handleInputChange}        
+                            />
+                        <SystemComponent></SystemComponent>
```

Comment:
Here, this empty SystemComponent, is where you should put your second textbox. Replace this SystemComponent with another FieldSection. Your third, fourth, fifth textbox should just be place one on top of another.

If this leads to layout problems in the form, please tell me.